code/game/dna/genes/speech.dm

```diff
@@ -225,7 +232,7 @@
 	addWordReplacement("beer","water with ice")
 	addWordReplacement("drink","water")
 	addWordReplacement("\\b(feminist|feminazi|SJW|social justice warrior)","empowered woman")
-	addWordReplacement("\\b(fuck you|i hate you|kill yourself|go die)","you're mean")
+	addWordReplacement("\\b(fuck you|i hate you|hug yourself|go die)","you're mean") //kill becomes hug
```

Comment:
I don't think this does what you meant

Reply:
eh, moving it before "kill" is filtered instead of changing it afterwards works too

Reply:
Ah, I see
Yeah, that's definitely a better solution